src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -234,7 +234,9 @@ protected int doWork() {
         final SAMFileHeader.SortOrder sortOrder = header.getSortOrder();
 
         final SAMFileHeader outputHeader = header.clone();
-
+        if(ASSUME_SORTED || ASSUME_SORT_ORDER != null) {
+            outputHeader.setSortOrder(SAMFileHeader.SortOrder.unsorted);
+        }
```

Comment:
Also `ASSUME_SORTED` means `coordinate`, so please either correct this or explain to me what's going on here.